### DISCUSSION TITLE
Update source code

### DISCUSSION THREAD

**Discussion by florianfish** [php/frankenphp#1084] [General]
Hello!

I'm new to using FrankenPHP. I managed to integrate it locally and in production, my website works without problem.
When I do a git pull nothing changes on the site. I tried rebuilding with no-cache but still nothing changed. I think I missed something but I can't see...

Thank you in advance,

**Comment by AlliBalliBaba** [comment]
Are you using FrankenPHP's worker mode? In worker mode file changes will not be reflected until Caddy is restarted. Alternatively it's also possible to watch a specific directory and [restart workers automatically](https://frankenphp.dev/docs/config/#watching-for-file-changes) when a file changes (be careful about using this option in prod though, you don't want to restart unnecessarily)
